cns/configuration/env.go

```diff
@@ -12,12 +12,20 @@ const (
 	// EnvNodeIP is the IP of the node running this CNS binary
 	EnvNodeIP = "NODE_IP"
 	// LabelSwiftV2 is the Node label for Swift V2
-	LabelSwiftV2 = "kubernetes.azure.com/podnetwork-multi-tenancy"
+	LabelSwiftV2   = "kubernetes.azure.com/podnetwork-multi-tenancy-enabled"
+	EnvPodCIDR     = "POD_CIDR"
+	EnvServiceCIDR = "SERVICE_CIDR"
```

Comment:
What about DNS?

Reply:
I didn't see DNS mentioned when we were discussing routing a few weeks ago, how/what would the route for DNS look like?